changelogs/unreleased/1093-DheerajSShetty

```diff
@@ -0,0 +1 @@
+Need to defer closing the the ReadCloser in ObjectStoreGRPCServer.GetObject
```

Comment:
Could you rename this file to `1236-DheerajSShetty` please? Our current format is to use the PR number.

Reply:
Ok